libs/onboarding-sdk-crypto/src/main/java/it/pagopa/selfcare/onboarding/crypto/config/LocalCryptoInitializer.java

```diff
@@ -2,26 +2,31 @@
 
 import it.pagopa.selfcare.onboarding.crypto.utils.CryptoUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 public class LocalCryptoInitializer {
 
+    private final static Logger log = LoggerFactory.getLogger(LocalCryptoInitializer.class);
+
     private LocalCryptoInitializer(){}
 
     public static LocalCryptoConfig initializeConfig() {
         LocalCryptoConfig config = new LocalCryptoConfig();
-        String cert = System.getProperty("crypto.key.cert");
-        String pKey = System.getProperty("crypto.key.private");
+        String cert = System.getenv("CRYPTO_CERT");
+        String pKey = System.getenv("CRYPTO_PRIVATE_KEY");
 
         try {
-            if(StringUtils.isBlank(cert) || StringUtils.isBlank(pKey)){
-                throw new IllegalStateException("Define private and cert values in order to perform locally sign operations");
-            }
+            if (StringUtils.isBlank(cert) || StringUtils.isBlank(pKey)) {
+                log.error("Define private and cert values in order to perform locally sign operations");
+            } else {
 
-            config.setCertificate(CryptoUtils.getCertificate(cert));
-            config.setPrivateKey(CryptoUtils.getPrivateKey(pKey));
+                config.setCertificate(CryptoUtils.getCertificate(cert));
+                config.setPrivateKey(CryptoUtils.getPrivateKey(pKey));
+            }
         } catch (Exception e) {
```

Comment:
this will return `config` anyway in case of exception, is this the expected behaviour?

Reply:
yes I want only to log if env variables are not present...